Sources/Clappr_iOS/Classes/Plugin/Core/QuickSeekCorePlugin.swift

```diff
@@ -23,6 +24,10 @@ public class QuickSeekCorePlugin: QuickSeekPlugin {
     }
     
     override func shouldSeek(point: CGPoint) -> Bool {
+        if chromelessMode {
```

Comment:
If we disable the gesture recognizers on `Core` (and therefore avoid triggering `didTappedCore` event), do we need this check here?

Reply:
I think a possible approach is to disable the `gestureRecognizers `on the core`s view because the QuickSeekCorePlugin creates another Gesture `doubleTapGesture` the adds to the core.
```
private func updateGestureRecognizers() {
        view.gestureRecognizers?.forEach{ $0.isEnabled = !chromelessMode }
}
```